notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,581 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "29b161d0-8144-4b3f-88fe-deb6abd21f1c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Meter Reader with PaddleOCR\n",
+    "This notebook shows how to create a meter reader with OpenVINO Runtime. We use the pre-trained [PP-OCR](https://github.com/PaddlePaddle/PaddleOCR) to build up a inference task pipeline:\n",
```

Comment:
Could you visualize the result image from each step in the codes? for example: a picture with edge bounding box. It will help our users to understand the workflow better.